nova/files/ocata/nova-controller.conf.Debian

```diff
@@ -2267,6 +2267,7 @@ force_dhcp_release=true
 # Reason:
 # nova-network is deprecated, as are any related configuration options.
 #dhcp_domain=novalocal
+dhcp_domain={{ controller.dhcp_domain }}
```

Comment:
You should provide default value to be backward compatible.
Also apply this change for all openstack versions.

Reply:
How should I define the default value here? Btw. default-value is set in the system level reclass model, which is "novalocal"

Reply:
Formula does not care about Reclass, nor system or any other level. map.jinja and rendered pillar is all that matters here.

You need to use at least this to be backward compatible:
```{{ controller.dhcp_domain|default('novalocal') }}```